http://trac.macports.org/ticket/47567
https://github.com/wxWidgets/wxWidgets/commit/f97112a21e0aadbec49df603f554ebf0088e4207

 Opening the font dialog would result in an exception in
 RunMixedFontDialog:
 
 -[NSView resetFlags]: unrecognized selector sent to instance 0x6080001285c0
 
 The code in question assumed that NSFontPanel's accessory view either
 didn't exist or was created by wx, and casted it to
 wxMacFontPanelAccView* without checking. But this assumption is no
 longer true on OS X 10.10, the view is apparently pre-set to some
 default NSView instance.
 
 Fix the code to check accessoryView's class before treating it as
 wxMacFontPanelAccView.

--- src/osx/carbon/fontdlgosx.mm.orig
+++ src/osx/carbon/fontdlgosx.mm
@@ -162,6 +162,6 @@
     [[fontPanel standardWindowButton:NSWindowCloseButton] setEnabled:NO] ;
 
-    wxMacFontPanelAccView* accessoryView = (wxMacFontPanelAccView*) [fontPanel accessoryView] ;
-    if ( accessoryView == nil)
+    wxMacFontPanelAccView* accessoryView = nil;
+    if ( [fontPanel accessoryView] == nil || [[fontPanel accessoryView] class] != [wxMacFontPanelAccView class] )
     {
         NSRect rectBox = NSMakeRect( 0 , 0 , 192 , 40 );
@@ -171,4 +171,8 @@
 
         [fontPanel setDefaultButtonCell:[[accessoryView okButton] cell]] ;
+    }
+    else
+    {
+        accessoryView = (wxMacFontPanelAccView*)[fontPanel accessoryView];
     }
 
